test/builtin/numbers/test_linalg.py

```diff
@@ -211,6 +211,12 @@ def test_inverse(str_expr, str_expected, fail_msg, warnings):
             "SingularValueDecomposition[{1, {2}}]",
             None,
         ),
+        (
+            "A = Array[a, {2,2}]; Eigenvalues[A.ConjugateTranspose[A]] // FullSimplify",
```

Comment:
@rocky, the example is already for a 2x2 matrix.

Reply:
I am sorry for the noise then and that I did not noticed previously.

Reply:
In any case, I split the test case in two parts, the first that computes the eigenvalues, and the second, that simplifies just the first eigenvalue.